### DISCUSSION TITLE
Documentation for lr-headless-modal

### DISCUSSION THREAD

**Discussion by alexbjorlig** [uploadcare/file-uploader#511] [Q&A]
Hi there.

Where do I find some documentation for the `lr-headless-modal `?

I sometimes get `CSS Data error: --cfg-accept` and `CSS Data error: --cfg-img-only` warnings, and having a hard time debugging why.

**Comment by nd0ut** [comment]
Hey @alexbjorlig,

we don't have such a block in our codebase. Could you please provide some insight into where you came across this block and which version of blocks you are using?

**Reply by alexbjorlig** [reply]
Hi there - thanks for the answer.

Ok - what is your recommended approach for doing "headless", i.e. open on the click of a button?

**Reply by nd0ut** [reply]
Here is an example of how to open uploader modal using custom button: https://codesandbox.io/s/file-uploader-custom-button-whfmgw?file=/index.html:445-464
